extensions/interactions/ItemSelectionInput/directives/ItemSelectionInputValidationServiceSpec.js

```diff
@@ -1,4 +1,4 @@
-// Copyright 2014 The Oppia Authors. All Rights Reserved.
+// Copyright 2019 The Oppia Authors. All Rights Reserved.
```

Comment:
Oops missed this one. Again, no need to edit!

Reply:
Done! PTAL.